test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerMockTest.java

```diff
@@ -446,7 +448,8 @@ private void signInNewSession(boolean isConsented, Roles role) throws Exception
 
         verify(cacheProvider).updateRequestInfo(requestInfoCaptor.capture());
         verify(mockResponse).setCookie(BridgeConstants.SESSION_TOKEN_HEADER, session.getSessionToken(),
-                BridgeConstants.BRIDGE_SESSION_EXPIRE_IN_SECONDS, "/");
+                BridgeConstants.BRIDGE_SESSION_EXPIRE_IN_SECONDS, "/",
+                BridgeConfigFactory.getConfig().get("webservices.url"), false, false);
```

Comment:
I think our build runs in dev, so this will fail in Travis. We should probably mock the env to dev, kinda like how we do it in BaseControllerTest. (2 places in AuthenticationControllerMockTest and 1 in BaseControllerTest.)